File: app/src/main/java/com/derlados/computerconf/PageFragment/InfoFragment.java
package com.derlados.computerconf.PageFragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.derlados.computerconf.R;

public class InfoFragment extends PageFragment {
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View fragment = inflater.inflate(R.layout.fragment_info, container, false);
        return fragment;
    }
}

File: app/src/main/java/com/derlados/computer_conf/views/InfoFragment.kt
package com.derlados.computer_conf.views

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.derlados.computer_conf.R

class InfoFragment: Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        return inflater.inflate(R.layout.fragment_info, container, false)
    }
}